p2p/pkg/notifications/notifications.go

```diff
@@ -0,0 +1,66 @@
+package notifications
+
+import (
+	"github.com/cskr/pubsub/v2"
+)
+
+const (
+	TopicPeerConnected    = "peer_connected"
+	TopicPeerDisconnected = "peer_disconnected"
+)
+
+// Notification is a struct that represents a notification. It has a Topic field
+// that represents the topic of the notification and a Value field that represents
+// the value of the notification. The Value field is a map[string]any, which means
+// that it can be any type.
+type Notification struct {
+	Topic string
```

Comment:
This fields shouldn't be exported so they cannot be modified outside of the package.